tower-http/src/request_id.rs

```diff
@@ -215,6 +215,12 @@ impl From<HeaderValue> for RequestId {
     }
 }
 
+impl fmt::Display for RequestId {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "{}", self.0.to_str().unwrap_or_default())
```

Comment:
This will be very misleading if the value isn't convertible to string so this approach won't work

Reply:
Can uuid not be converted into a string?

Reply:
Sure `Uuid` can, but `RequestId` can be [created from any `HeaderValue`](https://github.com/tower-rs/tower-http/blob/master/tower-http/src/request_id.rs#L197) which might not be valid utf-8.

Reply:
Considering business attributes, request-id is a string in most cases. Otherwise the meaning of the display cannot be found

Reply:
If the request-id is not composed of ascii characters, it is not a request-id.because it doesn't make any sense

Reply:
I don't agree with that. Non-ascii request ids are fine to uniquely identify a request. You just need the id to be unique.